static/app/components/events/interfaces/nativeFrame.tsx

```diff
@@ -91,7 +91,6 @@ function NativeFrame({
   isShowFramesToggleExpanded,
   isSubFrame,
   onShowFramesToggle,
-  isExpanded,
```

Comment:
This should also be removed from the `Props` type above. Any callsites that are passing it in will have to be updated as well.

Reply:
I will take a look and make required changes for it.